data/presets/playground/map.json

```
@@ -0,0 +1,19 @@
+{
+    "icon": "temaki-compass",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "fields": [
+        "subject",
+        "surface"
+    ],
+    "moreFields": [
+        "colour",
+        "subject/wikidata"
+    ],
```

What do you think about switching the priority of the `subject` and `subject/wikidata` fields around? Currently there fewer cases of `playground=map` objects which have `subject` but not `subject:wikidata` compared to the other way around. Also, in a `subject:wikidata` it's less easy to make typos, spelling mistakes, etc.

```suggestion
    "fields": [
        "subject/wikidata",
        "surface"
    ],
    "moreFields": [
        "colour",
        "subject"
    ],
```